jans-config-api/server/src/main/java/io/jans/configapi/service/auth/AssetService.java

```diff
@@ -152,16 +149,22 @@ public Document saveAsset(Document asset, InputStream documentStream) throws Exc
         ByteArrayOutputStream bos = getByteArrayOutputStream(documentStream);
         log.trace("Asset ByteArrayOutputStream :{}", bos);
 
-        // update asset revision
-        updateRevision(asset);
-
+        //get asset
+        asset = setAssetContent(asset, new Base64InputStream(getInputStream(bos), true));
```

Comment:
@pujavs shouldn't we close input stream ? Not closed steams can cause unreleased resources.

Reply:
@yuriyz have fixed the issue regarding stream closure

Reply:
Thanks, approved